lib/middleware/administration.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2014 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of fiware-orion-pep
+ *
+ * fiware-orion-pep is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * fiware-orion-pep is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with fiware-orion-pep.
+ * If not, seehttp://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License
+ * please contact with::[iot_support@tid.es]
+ */
+
+'use strict';
+
+var packageInformation = require('../../package.json'),
+    config = require('../../config');
+
+function retrieveVersion(req, res, next) {
+    res.status(200).json({
+        version: packageInformation.version,
```

Comment:
Do packageInformation object "maps" the information in package.json?

(Just curiosity)